charts/spire/crds/spire.spiffe.io_clusterfederatedtrustdomains.yaml

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
I think this is the filename causing the lint failure.

probably caused by `spire.spiffe.io…`

Reply:
I've changed the names of the CRD files.